### PR TITLE
Schedule Call AM/PM clock

### PR DESCRIPTION
**Pull request by peterp**
screen shot 2016-09-29 at 16 23 39

screen shot 2016-09-29 at 16 24 46

### CODE REVIEW

**Comment by peterp** [comment]
@justinvdm ready for review

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "app",
-  "version": "0.1.4",
+  "version": "0.1.5",
```
Comment (justinvdm) [inline package.json]: I'd rather we did version bumping using a release branch, or is this a commit that somehow managed to appear in the diff but isn't for this PR?

**Comment by justinvdm** [comment]
👍 !

File: package.json (hunk 1 above)
Comment (peterp) [inline package.json]: Yup. I don't think `git flow` pushes the develop branch after it's merged. 🍤

**Pull request merged**